### PR TITLE
Specify class for manual uploads and definitions

### PR DESCRIPTION
**Pull request by noah-tutt-praetorian**
### Summary

Specifies `class:manual` for regular uploads and `class:definition` for definition uploads.

### Type

New Feature

### Context

Implementation of https://github.com/praetorian-inc/chaos/pull/914

### CODE REVIEW

File: praetorian_cli/handlers/add.py [hunk 1]
```diff
@@ -23,9 +23,9 @@ def seed(controller, seed, status, comment):
 @add.command('file')
 @click.argument('name')
 @cli_handler
-def upload(controller, name):
+def upload(controller, name, clss):
```
Comment (peter-kwan) [inline praetorian_cli/handlers/add.py:26]: Also need to add a Click option. We should make that a required=True option and have choices for them

**Review by peter-kwan — CHANGES_REQUESTED** [review]

File: praetorian_cli/handlers/add.py (hunk 1 above)
Comment (noah-tutt-praetorian) [reply praetorian_cli/handlers/add.py:26]: ah actually, it shouldn't be an option, this is vestigal

**Review by Ayushi-12 — APPROVED** [review]
Re: Peter's earlier comment. 
We can skip adding a class option for now if manual/definitions are the only two.

**Comment by peter-kwan** [comment]
> Re: Peter's earlier comment. We can skip adding a class option for now if manual/definitions are the only two.

Even better!

**Review by peter-kwan — APPROVED** [review]

**Pull request merged**